server/types/express.d.ts

```diff
@@ -24,7 +25,7 @@ declare global {
       };
       userToken?: UserToken;
       personalToken?: PersonalToken;
-      loaders: Record<string, DataLoader>;
+      loaders: Record<keyof typeof models, DataLoader>;
```

Comment:
This is not necessarily true. `context.loaders.Contributors` for example doesn't relate to a DB model. In the past, FX rate loaders were not either. Loaders debounce expensive async operations and, most of the time, that's indeed meant to be used with DB models. But there are legit cases of using them with 3rd party APIs or other async calls as well.

Ideally, we'll type `loaders.index` and that will solve everything. Until them, I'll suggest we keep the existing

```suggestion
      loaders: Record<string, DataLoader>;
```